specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -5954,6 +6065,35 @@
       "title": "AtlasObjectId",
       "description": "Reference to an object-instance of an Atlas type - like entity."
     },
+    "PurviewObjectId": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AtlasObjectId"
+        },
+        {
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "displayText": {
+              "type": "string"
+            },
+            "itemPath": {
+              "type": "string"
+            },
+            "resourceId": {
+              "type": "string"
+            },
+            "properties": {
+              "type": "object",
+              "additionalProperties": {}
+            }
```

Comment:
If this is our data, do we really don't know what properties there is?

Reply:
Thanks. These are all our data. As PurviewObjectId is shared in many scenarios, it will contain different keys in various scenarios.